fs/procfs/Kconfig

```diff
@@ -31,15 +31,15 @@ menu "Exclude individual procfs entries"
 
 config FS_PROCFS_EXCLUDE_PROCESS
 	bool "Exclude process information"
-	default n
+	default DEFAULT_SMALL
```

Comment:
Hi @xiaoxiang781216 I saw that sched/Kconfig and fs/Kconfig has this already, but I suggest using "default y DEFAULT_SMALL" for legibility. I will submit a patch to fix sched and fs

Reply:
But, I think `default DEFAULT_SMALL` is more clear than `default y if DEFAULT_SMALL`. For example, what's default value if DEFAULT_SMALL disable?

Reply:
I think the opposite, ```default DEFAULT_SMALL``` give the impression that it is setting DEFAULT_SMALL. The ```default n``` is implicit when DEFAULT_SMALL is not enabled.